qa/workunits/rbd/rbd_mirror_group_simple.sh

```diff
@@ -3677,6 +3677,67 @@ test_demote_snap_sync()
   check_daemon_running "${secondary_cluster}"
 }
 
+declare -a test_demote_snap_sync_after_restart_1=("${CLUSTER2}" "${CLUSTER1}" "${pool0}" "${image_prefix}" 'wait_after_restart' 2)
+
+test_demote_snap_sync_after_restart_scenarios=1
+
+test_demote_snap_sync_after_restart()
+{
+  local primary_cluster=$1 ; shift
+  local secondary_cluster=$1 ; shift
+  local pool=$1 ; shift
+  local image_prefix=$1 ; shift
+  local scenario=$1 ; shift
+  local image_count=$(($1*"${image_multiplier}")) ; shift
+  local image0=test_image0
```

Comment:
This could be replaced with `${image_prefix}0` for consistency with other tests.